data/PA/incentives.json

```diff
@@ -0,0 +1,1023 @@
+      "rebate"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50,
+      "maximum": 50
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$50 rebate for ENERGY STAR certified clothes dryer."
+    },
+    "start_date": "2021",
+    "end_date": "2026"
+  },
+  {
+    "id": "PA-25",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75,
+      "maximum": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 rebate for ENERGY STAR certified clothes dryer for income-qualified residents."
+    },
+    "start_date": "2021",
+    "end_date": "2026",
+    "low_income": "pa-first-energy"
+  },
+  {
+    "id": "PA-26",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "weatherization",
+    "program": "pa_firstEnergy_wARMProgram",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Free energy audit and weatherization services for income-qualified residents."
+    },
+    "low_income": "pa-first-energy-warm"
+  },
+  {
+    "id": "PA-27",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 350
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $350 rebate for an in-home audit for electric heating and/or central A/C."
+    }
+  },
+  {
+    "id": "PA-28",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Free virtual home energy assessment. Assessment comes with a free energy savings kit mailed to home."
+    }
+  },
+  {
+    "id": "PA-29",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
```

Comment:
This is `heat_pump_air_conditioner_heater` I think? I forget where we stand on incentives for equipment that only cools and doesn't heat, but if we want this to show up, it should have that `item` value.